pkg/apis/sources/v1alpha1/couchdbsource_defaults_test.go

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
wrong file pushed? is this couchdb stuff?

Reply:
I have yet to do the tests, will change!